### ISSUE TITLE
Can not use with eslint-plugin-import

### ISSUE THREAD

**Issue by HsuTing** [tleunen/eslint-import-resolver-babel-module#85]
I try to use `eslint-plugin-import` with `eslint-import-resolver-babel-module`.
This can not work.

- `eslint-import-resolver-babel-module`: ^4.0.0 | ^5.0.0-beta.0
- `eslint-plugin-import`: ^2.11.0
- `babel-core`: ^7.0.0-0

When I use `eslint`, and `.eslintcache` does not exit.
It show:
```js
[eslint-import-resolver-babel-module] TypeError: Cannot read property 'find' of undefined
```
I found the problem is [here](https://github.com/tleunen/eslint-import-resolver-babel-module/blob/master/src/index.js#L15).

**Comment by HsuTing** [comment]
Oh, my fault.
I found the problem. I had a filename `eslint.js`.
Sorry to disturb you.

**Comment by HsuTing** [comment]
I still get `[eslint-import-resolver-babel-module] TypeError: Cannot read property 'find' of undefined`, when `import/no-unresolved` is detected.

```
[eslint-import-resolver-babel-module] TypeError: Cannot read property 'find' of undefined
    at result.plugins.filter.plugin (**/node_modules/eslint-import-resolver-babel-module/lib/index.js:20:50)
```

`eslint` does work, but I think this error should not be shown.

**Comment by jasonleibowitz** [comment]
@HsuTing Found a workaround from #67. The latest version of this library is 5.0.0-beta.0. If you update this library to that version this issue is resolved.

I'm not sure why the default version in npm is 4.0.0. Probably because v5 is marked as beta.

**Comment by HsuTing** [comment]
Oh! Sorry, I found the solution. If using `eslint-import-resolver-babel-module` with `eslint-plugin-import`, we should set like this:

```js
module.exports = {
  ...
  settings: {
    'import/resolver': {
      node: {},  // need to add this
      'babel-module': alias,
    },
  },
  ...
};
```

@jasonleibowitz  Thank for your answer!
